CGPs/cgp-0031.md

```diff
@@ -16,27 +16,112 @@ This proposal sets the initial parameters for and effectively activates Granda M
 
 There are a number of configurable parameters in `GrandaMento.sol`:
 
-1. **`address public approver;`** - Intended to be a multisig and has the authority to approve exchanges. The signers for the multisig are at the discretion of Governance, but it would comprise of community members or members on behalf of organizations that are aligned with the Celo network/ecosystem.
-  * This is proposed to be a 3 of 5 multisig. Details on proposed members TBD.
+1. **`address public approver;`** - Has the authority to approve exchanges.
+  * This is proposed to be a 4 of 7 multisig of community volunteers that are active within the Celo ecosystem. The multisig has an address of `0x0....0` and has the following signers:
```

Comment:
I assume that address has to be updated

Reply:
It can include the command to verify that that address indeed has a multisig (if we have one)

Reply:
Correct, yeah good idea

Reply:
I'm not totally aware that the 6 month term thing is something that's been discussed, so for now I just added a sentence as a reminder that this can be changed in the future at the discretion of governance. Personally I don't think it's worth having a process to explicitly create a new multisig in X months